ext/php7/engine_hooks.c

```diff
@@ -57,14 +58,12 @@ static zend_op_array *_dd_compile_file(zend_file_handle *file_handle, int type)
 }
 
 static void _compile_minit(void) {
-    if (get_dd_trace_measure_compile_time()) {
-        _prev_compile_file = zend_compile_file;
-        zend_compile_file = _dd_compile_file;
-    }
+    _prev_compile_file = zend_compile_file;
+    zend_compile_file = _dd_compile_file;
```

Comment:
Let's punt on it for this PR, but before this ships we should audit every zend fptr we override to make sure they all do an "are we enabled?" check before doing their work.